src/crimson/os/cyan_store.cc

```diff
@@ -0,0 +1,31 @@
+#include "cyan_store.h"
+
+#include <fmt/format.h>
+
+#include "common/safe_io.h"
+
```

Comment:
I didn't find where osd::mkfs is executed, if it executed in seastar world, here write_meta is write to file system we should use seastar write file call to avoid block.

Reply:
i haven't hooked it up with main.cc. the idea is to execute it when `--mkfs` is passed to crimson-osd, so we don't need existing osd to prepare the object store for us. if you think we should drop this commit in this PR, i can do so.

Reply:
> write_meta is write to file system we should use seastar write file call to avoid block

i am trying to reuse existing code for the non-critical task as much as possible, instead of rewriting it from scratch. we can always rework this part later on.